test/trento_web/views/v1/sap_system_view_test.exs

```diff
@@ -2,15 +2,18 @@ defmodule TrentoWeb.V1.SapSystemViewTest do
   use TrentoWeb.ConnCase, async: true
 
   import Phoenix.View
-
+  import Trento.Factory
   alias Trento.Databases.Projections.DatabaseInstanceReadModel
   alias Trento.SapSystems.Projections.SapSystemReadModel
 
   test "should add the system replication status to the secondary instance and should remove it from the primary one" do
+    database = insert(:database)
```

Comment:
Use `build`. The insert will create a big mess otherwise.
In general, don't use `insert` unless you using the `DataCase` template.
Without this, the database is not sandboxed and you will start seeing a lot of errors